python/ray/data/_internal/logical/operators/from_items_operator.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any, List
+
+from ray.data._internal.logical.operators.map_operator import AbstractMap
+
+
+class FromItems(AbstractMap):
```

Comment:
It's not really a map right? Shouldn't this inherit from Read?

Reply:
Yeah you're correct, it's not a map here. Discussed with @c21 earlier this week, and I think the distinction we wanted between `read_xxx` and `from_xxx` was that `read` is done from an external source, while `from` is read from in-memory. Should we still inherit from `Read`? As an alternative, we could instead inherit from the general parent `LogicalOperator` class?

Reply:
Yeah, if it's hard to represent as a Read, it seems ok to inherit from logical operator directly.